docs/docs/api_reference.md

```diff
@@ -55,6 +55,7 @@ There are some special variables that is available on the templating system:
 | `TASK` | The name of the current task. |
 | `ROOT_DIR` | The absolute path of the root Taskfile. |
 | `TASKFILE_DIR` | The absolute path of the included Taskfile. |
+| `WORKING_DIR` | The absolute path of the directory the command was run from. Use this instead of `PWD` as it will work on any OS. |
```

Comment:
> Use this instead of `PWD` as it will work on any OS.

I think `WORKING_DIR` is actually different from `PWD`? `PWD` will contain the directory the task in running in, and it _should_ work even on Windows given `mvdan/sh` handle it IIRC.

`WORKING_DIR`, on the other hand, contains the directory the user is calling Task from. The idea to add this is great, but I believe that we might want to rename to something different to avoid users confounding it with `PWD`.

Not sure about the name, but maybe `RUNNING_FROM_DIR`, `USER_WORKING_DIR`, or something else (naming things is hard 😅 ).

Reply:
Just sharing ideas.. What about `EXECUTION_DIR` or `INVOCATION_DIR`? 
btw. this is the feature we need to get started with taskfile :)

Reply:
@andreynering Apologies for the delayed reply. I was on holiday last week 🌴 

> I think `WORKING_DIR` is actually different from `PWD`

Yeah, the docs are definitely wrong here. I'll give this another go

> it [PWD] should work even on Windows

The builtin command `pwd` works on Windows but `PWD` (the environment variable) does not.

> Not sure about the name

I think `USER_WORKING_DIR` is sensible.

@TheKangaroo The problem with `EXECUTION_DIR` or `INVOCATION_DIR` is that it doesn't specify if its the directory where the _user_ is executing/invoking the Taskfile, or the directory where _Taskfile_ is executing/invoking commands. This is why I believe that something with `USER` in it is preferred. The naming for this is definitely a bit of a pain.

We could add `TASKFILE_WORKING_DIR` as well to try and clear up the distinction and just say that this is the same as `PWD` in the docs?